torch/testing/_internal/custom_tensor.py

```diff
@@ -0,0 +1,69 @@
+# mypy: ignore-errors
+
+import torch
+import torch.utils._pytree as pytree
+from torch.utils._python_dispatch import return_and_correct_aliasing
+
+
+# A simple tensor subclass that holds a tensor with custom metadata and custom method
+class CustomTensor(torch.Tensor):
```

Comment:
actually... is this subclass used anywhere in this PR?

Reply:
Yep it is used inside test_aotdispatch.py